code/datums/supplypacks/medical.dm

```diff
@@ -119,7 +119,7 @@
 			/obj/item/weapon/storage/belt/medical,
 			/obj/item/device/radio/headset/heads/cmo,
 			/obj/item/clothing/under/rank/chief_medical_officer,
-			/obj/item/weapon/reagent_containers/hypospray,
+			/obj/item/weapon/reagent_containers/hypospray/vial,
```

Comment:
This is the CMO crate in Cargo, the one that has the same contents as the CMO locker in Medical (same as how all the other job crates in Cargo have the same contents as the jobs in question) - changing this would make it inconsistent, I'd think?